TACC/Core-CMS-Custom
# feat: epoc_assets

## Overview

Add [EPOC](https://epoc.global/) assets.

## Related

- tested with https://github.com/TACC/Core-CMS/pull/856
- tested with https://github.com/TACC/Core-Portal-Deployments/pull/46

## Changes

- **added** `epoc_assets` dir with images

## Testing

1. Deploy to https://pprd.epoc.tacc.utexas.edu.
2. Compare to https://epoc.global/.
3. Verify branding is unchanged.
4. Verify logo is unchanged.
5. Verify favicon matches TACC favicon.
6. Verify page styles are unchanged.

## UI

### Banner

| Before (Prod) | After (Test) |
| - | - |
| <img width="960" alt="epoc prod home banner" src="https://github.com/user-attachments/assets/e46c2466-91fe-4039-9eac-e534b7726013"> | <img width="960" alt="epoc test home banner" src="https://github.com/user-attachments/assets/97f4f995-10fe-48ae-b333-8caaac7aba1f"> |

### Page Content

| Before (Prod) | After (Test) |
| - | - |
| ![epoc prod home](https://github.com/user-attachments/assets/1c92bf89-5913-4600-a797-4d031ccb12b8) | ![epoc test home](https://github.com/user-attachments/assets/c427d830-e3fd-44c4-8e2c-ca101a515518) |

### Favicon

| Before (Prod) | After (Test) |
| - | - |
| <img width="485" alt="epoc test home favicon" src="https://github.com/user-attachments/assets/77409732-7340-4cbc-a40a-c84a58c7b43b"> | <img width="485" alt="epoc prod home favicon" src="https://github.com/user-attachments/assets/37813cc0-3c91-4171-a6c1-162bac8fcf0d"> |
